src/GoogleFit/FitOnFhir.GoogleFit.Tests/QueueServiceTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Azure;
+using Azure.Storage.Queues;
+using Azure.Storage.Queues.Models;
+using Azure.Storage.Sas;
+using FitOnFhir.Common.Interfaces;
+using FitOnFhir.Common.Services;
+using FitOnFhir.Common.Tests.Mocks;
+using FitOnFhir.GoogleFit.Common;
+using FitOnFhir.GoogleFit.Tests.Mocks;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using NSubstitute;
+using NSubstitute.ExceptionExtensions;
+using NSubstitute.ReturnsExtensions;
+using Xunit;
+using QueueMessage = FitOnFhir.Common.Models.QueueMessage;
+
+namespace FitOnFhir.GoogleFit.Tests
```

Comment:
These tests should be in the FitOnFhir.Common.Tests

Reply:
I suppose they could be there.  The `ExpectedPlatformName` and `ExpectedPlatformUserId` are set to values that make sense for Google Fit, so there is a slight lean towards a particular platform.

Reply:
The `ExpectedPlatformName' and 'ExpectedPlatformUserId' can literally be any string, and should not be platform specific. The QueueService is platform agnostic and should be tested as such.